nibabel/funcs.py

```diff
@@ -115,15 +117,19 @@ def concat_images(images, check_affines=True):
     affine = img0.affine
     header = img0.header
     out_shape = (n_imgs, ) + i0shape
```

Comment:
Why is this less memory efficient?  It doesn't copy anywhere.  I assume that, under the covers, numpy preallocates a large array and does a memcopy.  So I believe this should be very efficient.